app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -21,3 +21,22 @@ end
 child :host_collections => :host_collections do
   attributes :id, :name
 end
+
+attributes :facts
+
+# All of the following requires another API call to fetch from Candlepin
+attributes :installedProducts
+attributes :system_type => :type
+attributes :compliance
+
+if @resource.respond_to?(:virtual_guest) || @resource.respond_to?(:virtual_host)
+  if @resource.virtual_guest
+    node :virtual_host do |system|
+      system.virtual_host.attributes if system.virtual_host
+    end
+  else
+    node :virtual_guests do |system|
+      system.virtual_guests.map(&:attributes)
+    end
+  end
+end
```

Comment:
I dont' think the content facet object currently provides this virtual host/guest functionality?